src/guess_density.jl

```diff
@@ -116,6 +116,28 @@ function gaussian_superposition(basis::PlaneWaveBasis{T}, gaussians) where {T}
     irfft(basis, ρ / sqrt(basis.model.unit_cell_volume))
 end
 
+"""
+    numeric_superposition(basis::PlaneWaveBasis)
+
+Build a superposition of functions on a radial grid as a guess for the density and
+magnetisation. Expects all the pseudopotentials of the atoms of the basis to contain
+atomic valence charge densities.
+"""
+function numeric_superposition(basis::PlaneWaveBasis{T}) where {T}
+    model = basis.model
+    ρ = zeros(complex(T), basis.fft_size)
+    for (iG, G) in enumerate(G_vectors(basis))
+        q = norm(model.recip_lattice * G)
```

Comment:
In our convention this should not be `q` (has no k component and lives on recip lattice). Maybe `Gnorm`?